src/win/fs.c

```diff
@@ -1497,13 +1500,27 @@ static void fs__fchmod(uv_fs_t* req) {
 }
 
 
-INLINE static int fs__utime_handle(HANDLE handle, double atime, double mtime) {
-  FILETIME filetime_a, filetime_m;
+INLINE static int fs__utime_handle(HANDLE handle,
+                                   double btime,
+                                   double atime,
+                                   double mtime) {
+  FILETIME filetime_b;
+  FILETIME filetime_a;
+  FILETIME filetime_m;
+
+  if (isnan(btime)) {
+    &filetime_b = NULL;
```

Comment:
Wouldn't this just be `filetime_b = NULL;`?

Reply:
Copy/paste fail, my bad.